### DISCUSSION TITLE
Zigbee network instability related to migration to SSB drive?

### DISCUSSION THREAD

**Discussion by mvtslot** [Koenkk/zigbee2mqtt#13590] [General]
Hi,

I have been using Zigbee2mqqt for long time without any problems, based on Home Assistant on RPi3b+, zstack USB stick and about 25 devices (routers and endpoint devices). 

Since sime weeks the network is very instable and need restarting every few ours. I checked and changes transmission power, surrounding WIFI networks, USB cables, etc. Nothing helped. 

Could this be related to a migration of my boot image from internal SD memory card to USB / SSD drive?

Thanks!

Marco

**Comment by Koenkk** [comment]
> Could this be related to a migration of my boot image from internal SD memory card to USB / SSD drive?


Yes, make sure the SSD is connected via a powered USB hub
